sdk/keyvault/azure-keyvault-secrets/src/main/java/com/azure/security/keyvault/secrets/SecretAsyncClient.java

```diff
@@ -408,6 +408,7 @@
      * <pre>
      * //Assuming secret is deleted on a soft-delete enabled vault.
      * {@codesnippet com.azure.keyvault.secrets.secretclient.getDeletedSecret#string}
+     * </pre>
```

Comment:
Same comments through here with pre tags being unnecessary and should be removed now.

Reply:
will remove both. 👍